sentry-apache-http-client-5/src/test/kotlin/io/sentry/transport/apache/ApacheHttpClientTransportTest.kt

```diff
@@ -120,4 +126,27 @@ class ApacheHttpClientTransportTest {
 
         verify(fixture.currentlyRunning, times(3)).decrement()
     }
+
+    @Test
+    fun `logs error when flush timeout was lower than time needed to execute all events`() {
+        val sut = fixture.getSut()
+        whenever(fixture.client.execute(any(), any())).then {
+            CompletableFuture.runAsync {
+                Thread.sleep(1000)
```

Comment:
Is there a way to reduce this to avoid slow builds? For example flush with 1ms instead of 500

Reply:
Done!